WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteViewModel.kt

```diff
@@ -738,15 +745,29 @@ class MySiteViewModel @Inject constructor(
     }
 
     fun onPullToRefresh() {
-        postCardsSource.refresh()
+        refresh()
     }
 
+    @Suppress("unused", "UNUSED_PARAMETER")
     fun onPostCardFooterLinkClick(postCardType: PostCardType) {
         selectedSiteRepository.getSelectedSite()?.let {
             // TODO: ashiagr implement navigation
         }
     }
 
+    fun shouldHideRefreshIndicator() = haveAllSourcesFinishedRefreshing()
+
+    private fun haveAllSourcesFinishedRefreshing(): Boolean {
```

Comment:
Suggestion (💡): What if this method was renaming to a simpler `areSourcesRefreshing()`, make public, and then this is used directly instead of having the `shouldHideRefreshIndicator()` intermediate?

Reply:
Sure. :)

Reply:
in c67b923